src/vm/core.zig

```diff
@@ -147,29 +151,32 @@ pub const CairoVM = struct {
         // Compute the destination address and get value from the memory.
         const dst_addr = try self.run_context.compute_dst_addr(instruction);
         const dst = try self.segments.memory.get(dst_addr);
-        _ = dst;
 
         // Compute the OP 0 address and get value from the memory.
         const op_0_addr = try self.run_context.compute_op_0_addr(instruction);
         // Here we use `catch null` because we want op_0_op to be optional since it's not always used.
-        const op_0_op = self.segments.memory.get(op_0_addr) catch null;
+        const op_0_op = try self.segments.memory.get(op_0_addr);
```

Comment:
are you sure of this ? it seems it should be allow to be null, with the `try` I think it breaks the implementation for some cases. worth checking

Reply:
I would keep it as a TODO because we still do not have the deduce op0 and compute op0 operands

Reply:
sounds good